Automattic/wp-calypso
# Whats New: Working in Atomic

Not urgent, may wait for all the changes done in https://github.com/Automattic/wp-calypso/pull/66407

#### Proposed Changes

* This PR reintroduces the `What's New Modal` in Atomic sites. It wasn't working before because of usual API problems in atomic sites (requests need to pass through Jetpack)
![image](https://user-images.githubusercontent.com/52076348/182814881-561551ad-e8ab-4772-a21a-987ad0a82936.png)
* Fixed the `useHasSeenWhatsNewModal` query in Atomic, used to determine if to show the dot next to the Help Center
![image](https://user-images.githubusercontent.com/52076348/182815614-6a0adda9-8f7a-464f-a189-8edede271164.png)
* Changed the tracking event for the modal to `calypso_whats_new_slide_view` from `wpcom_whats_new_slide_view`, because it was blocked as not following the guidelines.

#### Testing Instructions



* Checkout branch
* `yarn dev --sync` in `apps/editing-toolkit`
* Check that the query works in simple and atomic sites



Related to #65973 
Fixes #65973
